### ISSUE TITLE
Improve use of CKAN metadata on the About This Dataset page

### ISSUE THREAD

**Issue by meh9** [TerriaJS/terriajs#2544]
When adding this CKAN dataset to a Terria instance:

https://data.gov.au/dataset/native-title-determinations-national-native-title-register

The About This Dataset shows:

```
Service Contact

Department of the Prime Minister and Cabinet
data@pmc.gov.au

Data Custodian

The National Native Title Tribunal assists people to facilitate timely and effective native title outcomes through delivery of a wide range of services.
```

However, the data.gov.au page also has a Contact Point of

```
National Native Title Tribunal
geoservices@nntt.gov.au
```

It would be very useful to have that as the Data Custodian on the About This Dataset page.

There are also many other fields that would be very useful to have on the About This Dataset page, we should probably add most of these:

```
Title	Native Title Determinations
Type	Dataset
Language	English
Licence	Creative Commons Attribution 4.0 International
Data Status	active
Update Frequency	daily
Landing Page	http://data.gov.au/dataset/ecdbbb6c-c374-4649-9cd3-0677f44182c9
Date Published	2013-05-12
Date Updated	2017-05-29
Contact Point	
National Native Title Tribunal
geoservices@nntt.gov.au
Temporal Coverage	1994 to Current
Geospatial Coverage	Australia
Jurisdiction	Commonwealth of Australia
Data Portal	data.gov.au
Publisher/Agency	National Native Title Tribunal
```

**Comment by a-stacey** [comment]
@meh9 When you have a minute we should have a quick chat about the UI that you were thinking about here, are we just going to include specific attributes, or do we want to list all the meta data available?

I'm guessing that it's just specific attributes that you were thinking otherwise its probably going to become unruly and unhelpful very quickly.

Here is a screenshot of the proof of concept that I mocked up which just auto fills all the possible data:
![screenshot from 2017-07-26 15-30-09](https://user-images.githubusercontent.com/4453477/28605942-aafdcbf0-7217-11e7-9e82-12ad9c782f9b.png)

If we want to automate all of the metadata I'm assuming that we would go down the route of what the data source details, but I think thats not going to add much value since most users wont click through and see the additional information:
![screenshot from 2017-07-26 15-30-23](https://user-images.githubusercontent.com/4453477/28606008-2fbcb8c4-7218-11e7-8b3c-9466462c07cf.png)

**Comment by meh9** [comment]
Let's do these ones:

```
Licence	Creative Commons Attribution 4.0 International
Update Frequency	daily
Date Published	2013-05-12
Date Updated	2017-05-29 
Contact Point	National Native Title Tribunal geoservices@nntt.gov.au
Publisher/Agency	National Native Title Tribunal
```

If there is a collision (i.e. a key or field name that exists in both the CKAN and the service) then the CKAN should override the service as the user has the ability to change the CKAN metadata, but may not have access to change the service data (as in the data.gov.au case).

**Comment by a-stacey** [comment]
Have added the fields manually as shown:
![screenshot from 2017-08-01 16-47-17](https://user-images.githubusercontent.com/4453477/28812714-529524ec-76d9-11e7-9d16-469a73c2390d.png)

**Comment by kring** [comment]
This was addressed by #2614.
